src/components/common/Partners/PartnerBlock.jsx

```diff
@@ -2,21 +2,24 @@ import React from 'react';
 import { Link } from 'react-router';
 
 function PartnerBlock(props) {
+  const partner = props.item.attributes;
+  const imgPath = `${config.CMS_API_URL}${partner.logo.medium}`;
+
   return (
     <article className="c-partners-block column small-12 medium-6">
       <div className="logo-container">
-        <Link to={`/partners/${props.partner.name}`}>
-          <img src={require(`../../../../public/images/partners/${props.partner.img}`)} className="logo" />
+        <Link to={`/about/partners/${props.item.id}`}>
+          <img src={imgPath} className="logo" title={partner.name} />
         </Link>
       </div>
-      <p className="description c-text -extra-big">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate.</p>
-      <a href={props.partner.url} className="c-text -extra-big">Read more</a>
+      <p className="description c-text -extra-big">{partner.summary}</p>
+      <a href={partner.website !== '' ? partner.website : "#"} className="c-text -extra-big" target="_blank">Read more</a>
```

Comment:
I think that it would be better if we show only the link if partner.website is present, instead of leave a blank link there...